CHANGELOG.txt

```diff
@@ -47,6 +47,7 @@ Changelog
  * Choice blocks in StreamField now show label rather than value when collapsed (Jérôme Lebleu)
  * Added documentation to clarify configuration of user-uploaded files (Cynthia Kiser)
  * Fix: Accessibility fixes for Windows high contrast mode; Dashboard icons colour and contrast, help/error/warning blocks for fields and general content, side comment buttons within the page editor, dropdown buttons (Sakshi Uppoor, Shariq Jamil, LB (Ben Johnston), Jason Attwood)
+ * Change security contact address to security@wagtail.org (Jake Howard)
```

Comment:
I'd tend not to give this sort of documentation tweak a changelog entry... but if it's getting one, it should be above the 'Fix:' section and also included in https://github.com/wagtail/wagtail/blob/main/docs/releases/2.16.md.